CHANGELOG.md

```diff
@@ -240,6 +240,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
```

Comment:
Can you update this too?

```
$ grep -iIrn "period" docs
docs/source/common/weights_loading.rst:74:You can also control more advanced options, like `save_top_k`, to save the best k models and the `mode` of the monitored quantity (min/max), `save_weights_only` or `period` to set the interval of epochs between checkpoints, to avoid slowdowns.
```